internal/pkg/middleware/auth.go

```diff
@@ -35,15 +36,79 @@ func AuthMiddleware(next http.Handler) http.Handler {
 			return
 		}
 
+		helpers.WhiteListMutext.Lock()
+		_, ok := helpers.TokenList[tokenString]
```

Comment:
you are keeping token in memory.

what happen if service goes down for 2 min, all tokens were lost.

Reply:
as we discussed kept as it is